libs/clients/icelandic-health-insurance/rights-portal/src/lib/rightsPortalClient.config.ts

```diff
@@ -0,0 +1,25 @@
+import { defineConfig } from '@island.is/nest/config'
+import { z } from 'zod'
+
+const schema = z.object({
+  fetch: z.object({
+    timeout: z.number().int(),
+    scope: z.array(z.string()),
+  }),
+  xRoadServicePath: z.string(),
+})
+
+export const RightsPortalClientConfig = defineConfig<z.infer<typeof schema>>({
+  name: 'RightsPortalConfig',
+  schema,
+  load: (env) => ({
+    xRoadServicePath: env.required(
+      'XROAD_HEALTH_INSURANCE_MY_PAGES_PATH',
+      'IS-DEV/GOV/10007/SJUKRA-Protected/minarsidur',
+    ),
```

Comment:
Does this environment variable exist in the api infra file for all envs?

Reply:
Its from the HealthInsurance xroad config